### PR TITLE
Resolved sorting and order history issue

### PR DESCRIPTION
**Pull request by Abdulrehman53**
**Why**

Need to resolve sorting issue in category listing
Need to resolve reset issue in order history

**This PR resolve these issues :**

Resolve sorting issue in category listing
Resolve reset issue in order history

### CODE REVIEW

File: lib/screens/categorylisting.dart [hunk 1]
```diff
@@ -627,7 +629,7 @@ class _CategoryListingState extends State<CategoryListing> {
     });
     var response;
     print(sortBy);
-    if (sortBy != null) {
+    if (sortBy != null && sortBy.length > 0) {
```
Comment (gopalshimpi) [inline lib/screens/categorylisting.dart:632]: here `if(sortBy)` this would be sufficient.

File: lib/utils/constants.dart [hunk 2]
```diff
@@ -1,9 +1,8 @@
 class Settings {
```
Comment (gopalshimpi) [inline lib/utils/constants.dart]: Why did you point it to staging?

**Review by gopalshimpi — CHANGES_REQUESTED** [review]
@Abdulrehman53 Please go through the changes.

File: lib/screens/categorylisting.dart (hunk 1 above)
Comment (Abdulrehman53) [reply lib/screens/categorylisting.dart:632]: if(sortBy) is not sufficient. sortBy is not bool. I just wanted to make sure for a null check on sortBy.

File: lib/utils/constants.dart (hunk 2 above)
Comment (Abdulrehman53) [reply lib/utils/constants.dart]: To test order history. Revert back

**Review by gopalshimpi — APPROVED** [review]
LGTM 👍

**Pull request merged**